examples/server.rs

```diff
@@ -52,26 +52,28 @@ async fn handle_connection(peer_map: PeerMap, raw_stream: TcpStream, addr: Socke
 
     let (outgoing, incoming) = ws_stream.split();
 
-    let broadcast_incoming = incoming.try_for_each(|msg| {
-        println!(
-            "Received a message from {}: {}",
-            addr,
-            msg.to_text().unwrap()
-        );
-        let peers = peer_map.lock().unwrap();
-
-        // We want to broadcast the message to everyone except ourselves.
-        let broadcast_recipients = peers
-            .iter()
-            .filter(|(peer_addr, _)| peer_addr != &&addr)
-            .map(|(_, ws_sink)| ws_sink);
-
-        for recp in broadcast_recipients {
-            recp.unbounded_send(msg.clone()).unwrap();
-        }
-
-        future::ok(())
-    });
+    let broadcast_incoming = incoming
+        .try_filter(|msg| future::ready(!msg.is_close()))
```

Comment:
Should this have a comment because it's an `examples`? It's hard to sum this up in one line but something like
```suggestion
        .try_filter(|msg| {
          // If we broadcast the Close message from one client,
          // it will close the other connected clients.
          future::ready(!msg.is_close())
        })
```
or something along those lines

Reply:
Seems like a good idea to me, yes

Reply:
https://github.com/sdroege/async-tungstenite/pull/23/commits/2a3c60836c8aebb113d0e6197ba4749ebc8d8b3c

Would you like that squashed before merging?

Reply:
Yes that would be best. Thanks!